UnitTests/UnitTests/ProjectBLTests.cs

```diff
@@ -72,8 +102,61 @@ public static Mock<UserManager<ApplicationUser>> MockUserManager(List<Applicatio
         }
 
         [TestMethod]
-        public void TestMethod1()
+
+		public void TestGetProject()
+		{
+			
+			int projectId = 1;
+
+			Project project = ProjectBusinessLogic.GetProject(projectId);
+
+
+			Assert.AreEqual(projectId, project.Id);
+			
+		}
+
+        [TestMethod]
+        public void TestGetProject_IdIsNull()
         {
-        }
-    }
+			int? projectId = null;
+
+			Project project = ProjectBusinessLogic.GetProject(projectId);
+
+            Assert.IsNull(project);
+		}
+
+        [TestMethod]
+
```

Comment:
Test Method names should be more detailed.
"DeleteProject_DeleteCorrectProject"